src/Rest/Routes/AbstractUtilsBaseRoute.php

```diff
@@ -462,6 +488,9 @@ protected function getFormDetailsApi($request): array
 		// Populare files on upload. Only populated on file upload.
 		$output[UtilsConfig::FD_FILES_UPLOAD] = $this->prepareFile($request->get_file_params(), $params['params'] ?? []);
 
+		// Populare action.
```

Comment:
```suggestion
		// Populate action.
```
Please also apply to other instances